### ISSUE TITLE
Over fitting or not ?

### ISSUE THREAD

**Issue by thiziri** [faneshion/MatchZoo#132]
Hi,
@faneshion , @pl8787 , @bwanglzu 
I'm running the DRMM model on trec data set (Robust), I've got the following curves (with 5-cross validation):
![data_fqrls_reranklm_f0](https://user-images.githubusercontent.com/17499919/40612875-8b87d102-627c-11e8-935b-83e86f469bc6.png)
![data_fqrls_reranklm_f1](https://user-images.githubusercontent.com/17499919/40612876-8bbd589a-627c-11e8-962b-57d2d324f3a9.png)
![data_fqrls_reranklm_f2](https://user-images.githubusercontent.com/17499919/40612877-8bd95c48-627c-11e8-8438-8397422d5af8.png)
![data_fqrls_reranklm_f3](https://user-images.githubusercontent.com/17499919/40612878-8bf44878-627c-11e8-983f-e64bfe3d0704.png)
![data_fqrls_reranklm_f4](https://user-images.githubusercontent.com/17499919/40612879-8c13b19a-627c-11e8-9916-db723eaa3ae5.png)
Is that means that the model over fitts the data?
Thanks in advance

**Comment by faneshion** [comment]
I don't think this is overfitting as the performance of valid and test dataset is consistent. it seems that the model have converged from less than 100 iterations. You can try to increase the number of parameters to see if the model will overfit?

**Comment by thiziri** [comment]
Okay thanks.
You mean that I could try with more nodes per a layer? and more layers?
In fact the configuration is:
- 2 hidden layers with 5 and 1 unit respectively
- learning rate = 0.01
- Dropout : 0.5
Do you think it's okay ?

**Comment by faneshion** [comment]
Yeah, you can increase the number of layers or add more nodes per layer. The size of parameters dependents on your task and dataset.

**Comment by thiziri** [comment]
Thank you for your responses.
For a data set with more than 528K documents, totaling 253M words (more then 664K unique words), how many layers and nodes, or what parameter configuration should be better for training DRMM or another model in MatchZoo? What do you recommend?
Thanks again.
